dist/common/systemd/scylla-helper.slice

```diff
@@ -0,0 +1,27 @@
+[Unit]
+Description=Slice used to run companion programs to Scylla. Memory, CPU and IO restricted
+DefaultDependencies=no
+Before=slices.target
```

Comment:
Not recommended according to the documentation of slices.target.

Reply:
Sorry, what exactly is not recommended ? (default dependencies or before)

Can you point me to the docs ?

Reply:
Found it -> it is DefaultDependencies that is not recommended. I will remove.